samples/LoggingTracer/LoggingTracer.AspNetCore/wwwroot/js/site.js

```diff
@@ -0,0 +1,4 @@
+﻿// Please see documentation at https://docs.microsoft.com/aspnet/core/client-side/bundling-and-minification
```

Comment:
aren't those coming with some NuGet or other packaging scripts? It will be great to avoid the need to push those into our repo.

Reply:
obsolete since I use a different template now